Block/FormatterBlockService.php

```diff
@@ -23,7 +23,7 @@
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class FormatterBlockService extends BaseBlockService
+class FormatterBlockService extends AbstractBlockService
```

Comment:
@jordisala1991 Is this a typo? Did you mean to extend `Sonata\BlockBundle\Block\Service\AbstractAdminBlockService` instead?